source/globalCommands.py

```diff
@@ -1229,10 +1229,7 @@ def script_toggleVirtualBufferPassThrough(self,gesture):
 		focus = api.getFocusObject()
 		vbuf = focus.treeInterceptor
 		if not vbuf:
-			# #2023: Search the focus and its ancestors for an object for which browse mode is optional.
 			for obj in itertools.chain((api.getFocusObject(),), reversed(api.getFocusAncestors())):
-				if obj.shouldCreateTreeInterceptor:
-					continue

```

Comment:
Can you explain to me why it is no longer necessary to skip objects which a treeInterceptor should always be created?